edit/ls-colors.go

```diff
@@ -82,6 +92,17 @@ func parseLsColor(s string) *lsColor {
 			continue
 		}
 		key, value := words[0], words[1]
+		filterValues := []string{}
+		for _, splitValue := range strings.Split(value, ";") {
+			if splitValue == "0" || splitValue == "00" || splitValue == "000" {
```

Comment:
You should also check for "" here, which is equivalent to "0". I think it makes sense to use `strings.Count(s, "0") == len(s)`.